provisioner/influx-series.js

```diff
@@ -39,6 +39,78 @@ module.exports.spotRequestsSubmitted = new base.stats.Series({
   },
 });
 
+// Store the times when a spot request is fulfilled
+module.exports.spotRequestsFulfilled = new base.stats.Series({
```

Comment:
module.exports === exports, i thought.  I've always done module.exports...

use strict is just habbit and doesn't hurt i guess.